.annotations_sample

```diff
@@ -0,0 +1,22 @@
+source_path: /Users/brianmesick/Dev/devstack-docker/edx-platform
+report_path: reports
+annotations:
+    pii:
+        - ".. pii::"
+        - ".. pii_types::":
+            - id
+            - name
+            - other
```

Comment:
My preference here would be to explicitly call out that these are options/choices, like:

```
        - "..pii_types::":
            choices: [id, name, other]
```

...with the intention of making the config more self-documenting.